consensus/obcpbft/pbft-core_test.go

```diff
@@ -1296,3 +1295,59 @@ func TestNilCurrentExec(t *testing.T) {
 	p := newPbftCore(1, loadConfig(), &omniProto{})
 	p.execDoneSync() // Per issue 1538, this would cause a Nil pointer dereference
 }
+
+func TestNetworkNullRequests(t *testing.T) {
+	validatorCount := 4
+	net := makePBFTNetwork(validatorCount, func(pe *pbftEndpoint) {
+		pe.pbft.nullRequestTimeout = 200 * time.Millisecond
+		pe.pbft.requestTimeout = 500 * time.Millisecond
+	})
+	defer net.stop()
+
+	msg := createPbftRequestWithChainTx(1, 0)
+	net.pbftEndpoints[0].pbft.manager.queue() <- msg
+
+	go net.processContinually()
```

Comment:
`processContinually` seems to be a source of many of our race conditions in the tests.  Why not:

```
net.process()
// Test the current state
for {
    time.Sleep(1 * time.Second)
    net.process()
    // Test the current state and break on success, or fail on enough retries
}
```

Ultimately this should be cleaned up by the deterministic timer wheel stuff, but I still like this approach better because of the first pass testing.